README.md

```diff
@@ -19,8 +19,8 @@ Additionally, the tracker...
 
 ![](http://i.imgur.com/zG3eV8V.png)
 
-Download it [here](https://github.com/Hyphen-ated/RebirthItemTracker/releases) (get the latest file that doesn't have
-"source code" in the name).
+Download it [here](https://github.com/rchardon/RebirthItemTracker/releases) or [here](https://github.com/Hyphen-ated/RebirthItemTracker/releases) for version 3.0.0 and downwards 
```

Comment:
In a pull request, you shouldn't change references pointing to the main repository. This cannot be merged while this is present.